### ISSUE TITLE
GUAC_USER and GUAC_PASS seem to be disregarded

### ISSUE THREAD

**Issue by audech** [linuxserver/docker-calibre#17]
## Expected Behavior
If GUAC_USER and GUAC_PASS are set, I would expect to have to provide these credentials in order to connect to Guacamole.

## Current Behavior
When opening the website I am directly connected to Guacamole and the Calibre GUI.

## Steps to Reproduce
1. Set GUAC_USER and GUAC_PASS
2. Start container
3. Open configured port (8080 by default)

## Environment
**OS:**  Synology DSM 6.2.2-24922
**CPU architecture:** x86_64     
**How docker service was installed:** from Docker Hub

## Command used to create docker container (run/create/compose/screenshot)
<img width="751" alt="ls-calibre" src="https://user-images.githubusercontent.com/3726574/67162785-e7f0a680-f367-11e9-8c37-55b42e18d491.png">

## Docker logs
See attachment
[log.txt](https://github.com/linuxserver/docker-calibre/files/3748073/log.txt)

Thanks for your help!

**Comment by CHBMB** [comment]
Have you done this for `GUAC_PASS`

https://github.com/linuxserver/docker-calibre#application-setup

**Comment by theilenz** [comment]
I used

`echo -n password | openssl md5`

and set the md5 hash as password.

It is working fine for me.

**Comment by audech** [comment]
Ok, found it - I had only set the enironment variables after the first start of the container. I have now updated the image, cleared the container and started again: now it works.
This is probably evident to anyone knowing a bit more about the workings of Docker than me (that is most people...) - perhaps add something to the instructions for the likes of me?
Feeling embarassed now, sorry for bothering and thanks for your help!
